### PR TITLE
HIR-77: Change page titles

### PR DESCRIPTION
**Pull request by kuskido**

### CODE REVIEW

File: src/core/route.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import propTypes from 'prop-types';
+import { Route as ReactRoute } from 'react-router-dom';
+import tracker from 'matsumoto/src/core/misc/tracker';
+
+const Route = (props) => {
+    useEffect(() => {
+        document.title = (props.title ? `${props.title} - ` : '') + 'Property Management System';
```
Comment (alexander-plekhanov) [inline src/core/route.js:8]: Happytravel.com

File: src/core/route.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    });
+
+    return <ReactRoute {...props} />;
+};
+
+Route.propTypes = {
+    title: propTypes.string
+}
+
+export default Route;
```
Comment (alexander-plekhanov) [inline src/core/route.js:19]: don't need to import this file

File: src/core/routes.js [hunk 3]
```diff
@@ -11,44 +13,80 @@ import RoomPage from 'pages/room/room-page';
...
+            <Route
+                exact
+                path="/"
+                title={t('Accommodations')}
+                component={AccommodationsList}
+            />
+            <Route
+                exact
+                path="/accommodation/:accommodationId/room"
+                title={t('Create New Room')}
```
Comment (alexander-plekhanov) [inline src/core/routes.js:29]: Room creation

File: src/core/routes.js [hunk 4]
```diff
@@ -11,44 +13,80 @@ import RoomPage from 'pages/room/room-page';
...
+                component={RoomPage}
+            />
+            <Route
+                path="/accommodation/:accommodationId/room/:id"
+                title={t('Edit Accommodation Room')}
```
Comment (alexander-plekhanov) [inline src/core/routes.js:34]: Edit Room

File: src/core/routes.js [hunk 5]
```diff
@@ -11,44 +13,80 @@ import RoomPage from 'pages/room/room-page';
...
+                component={RoomPage}
+            />
+            <Route
+                path="/accommodation/:accommodationId/rooms"
+                title={t('Accommodation Rooms')}
```
Comment (alexander-plekhanov) [inline src/core/routes.js:39]: Rooms

File: src/core/routes.js [hunk 6]
```diff
@@ -11,44 +13,80 @@ import RoomPage from 'pages/room/room-page';
...
+                component={RoomsList}
+            />
+            <Route
+                exact
+                path="/accommodation"
+                title={t('Create Accommodation')}
```
Comment (alexander-plekhanov) [inline src/core/routes.js:45]: Accommodation Creation

File: src/core/routes.js [hunk 7]
```diff
@@ -11,44 +13,80 @@ import RoomPage from 'pages/room/room-page';
...
+            />
+            <Route
+                path="/contracts"
+                title={t('Contracts')}
+                component={ContractsList}
+            />
+            <Route
+                exact
+                path="/contract"
+                title={t('Create Contract')}
```
Comment (alexander-plekhanov) [inline src/core/routes.js:61]: Contract Creation

File: src/core/routes.js [hunk 8]
```diff
@@ -11,44 +13,80 @@ import RoomPage from 'pages/room/room-page';
...
+            />
+            <Route
+                exact
+                path="/contract/:id"
+                title={t('Edit Contract')}
+                component={ContractPage}
+            />
+            <Route
+                path="/contract/:id/seasons"
+                title={t('Edit Contracts Seasons')}
```
Comment (alexander-plekhanov) [inline src/core/routes.js:72]: Seasons

File: src/core/routes.js [hunk 9]
```diff
@@ -11,44 +13,80 @@ import RoomPage from 'pages/room/room-page';
...
+                component={SeasonsList}
+            />
+            <Route
+                path="/contract/:id/calendar"
+                title={t('Edit Contracts Calendar')}
```
Comment (alexander-plekhanov) [inline src/core/routes.js:77]: Season Calendar

File: src/core/route.js (hunk 2 above)
Comment (kuskido) [reply src/core/route.js:19]: I thought that we should to change postfix from `HappyTravel.com` to something else.

If we leave the same postfix then of course this file can be taken from matsumoto.

File: src/core/routes.js (hunk 6 above)
Comment (kuskido) [reply src/core/routes.js:45]: `Accommodation Creation`
but
`Edit Accommodation`?

This is allright?

File: src/core/route.js (hunk 2 above)
Comment (alexander-plekhanov) [reply src/core/route.js:19]: Don't need now. And also if sometime we do, it'll be better to make postfix configurable in the source file.

**Review by alexander-plekhanov — APPROVED** [review]

**Pull request merged**